owlbot-postprocessor/lib/owlbot/common_modifiers.rb

```diff
@@ -134,6 +134,26 @@ def prevent_overwrite_of_existing path, name: nil
       end
     end
 
+    ##
+    # A temporary convenience method that adds `ostruct` as required for test helpers files,
+    # which are located in `test/helper.rb`.
+    #
+    # This is only added for auto-generated files that do not have the dependency explicitly listed.
+    #
+    # @param name [String] Optional name for the modifier to add. Otherwise, a default is provided.
+    #
+    def require_open_struct_in_test_helper name: nil
+      path = [/test\/helper\.rb$/]
+      name ||= "require_open_struct_in_test_helper"
+      modifier path: path, name: name do |src, _dest|
+        if src&.match("Auto-generated by gapic-generator-ruby.") && !src&.match?(/require "ostruct"/)
```

Comment:
I think in both cases you can just say `src&.include?("the string")`. (And you don't need a full regex for the `require "ostruct"`.) The above calls to `match` and `match?` work, but do much more than you need.